test_cases/test_dojo.py

```diff
@@ -97,23 +100,136 @@ def test_print_unallocated_with_filename_unspecified(self):
+		self.the_dojo.add_person("Pete", "pete@pete", "staff")
+		self.the_dojo.add_person("Matt", "matt@matt", "staff")
+		self.the_dojo.add_person("Patt", "patt@patt", "fellow")
+		self.the_dojo.add_person("Faith", "faith@faith", "staff")
+		self.the_dojo.add_person("Fait", "fait@fait", "fellow", "y")
+		self.the_dojo.add_person("Fai", "fai@fai", "staff")
+		self.the_dojo.create_room("office", "Green")
+		self.the_dojo.add_person("Marie", "maz@maz", "fellow","y")
+		result = self.the_dojo.reallocate_person("maz@maz", "Red")
+		self.assertEqual(result, "The room Red is full! Cannot reallocate FELLOW-Marie")
+
+	def test_reallocate_person_can_reallocate_a_fellow(self):
+		self.the_dojo.create_room("office", "Red")
+		self.the_dojo.add_person("Josh", "josh@josh", "fellow", "y")
+		self.the_dojo.create_room("office", "Green")
+		room_occupancy = [room.occupants for room in self.the_dojo.all_offices\
+		 if room.name == "Green"]
+		count_before_reallocation = len(room_occupancy[0])
+		self.the_dojo.reallocate_person("josh@josh", "Green")
+		room_occupancy = [room.occupants for room in self.the_dojo.all_offices\
+		 if room.name == "Green"]
+		count_after_reallocation = len(room_occupancy[0])
+		self.assertEqual(count_after_reallocation - count_before_reallocation, 1, 
+			msg = "Reallocation was unsuccessful!" )
+
+	def test_reallocate_person_successfully_reallocates_fellow_to_livingspace(self):
+		self.the_dojo.create_room("office","Red")
+		self.the_dojo.create_room("livingspace", "Crimson")
+		self.the_dojo.add_person("Matt","matt@matt", "fellow", "y")
+		self.the_dojo.create_room("livingspace","Emerald")
+		room_occupancy = [room.occupants for room in self.the_dojo.all_livingspace\
+		 if room.name == "Emerald"]
+		count_before_reallocation = len(room_occupancy[0])
+		self.the_dojo.reallocate_person("matt@matt", "Emerald")
+		room_occupancy = [room.occupants for room in self.the_dojo.all_livingspace\
+		 if room.name == "Emerald"]
+		count_after_reallocation = len(room_occupancy[0])
+		self.assertEqual(count_after_reallocation - count_before_reallocation, 1, 
+			msg = "Reallocation was unsuccessful!" )
+
+	def test_load_people_successfully(self):
+		with open("LoadFile", "w+") as input_file:
+			input_file.write("RANDY RANDY@RANDY STAFF\n")
+			input_file.write("TYRESE TY@TY FELLOW Y")
+			input_file.write("HEATHER HEATH@HEATH FELLOW")
+
+		initial_staff_count = self.the_dojo.all_staff
+		initial_fellow_count = self.the_dojo.all_fellows
+		self.the_dojo.load_people("LoadFile")
+		final_staff_count = self.the_dojo.all_staff
+		final_fellow_count = self.the_dojo.all_fellows
+		self.assertEqual(final_staff_count - initial_staff_count, 1,
+			msg = "Error loading Staff")
+		self.assertEqual(final_fellow_count - initial_fellow_count, 2,
+			msg = "Error Loading Fellows")
+
+	def test_load_people_returns_error_if_file_doesnot_exist(self):
+		result = self.the_dojo.load_people("LoadFile")
+		self.assertEqual(result, "The file LoadFile doesnot exist!")
+
```

Comment:
Yes contents in the file
For instance, if the valid format is SIMON PATTERSON FELLOW Y an invalid format would be Y SIMON PATTERSON FELLOW